.github/workflows/adhoc_build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
             ext: ".exe"
           - os: macos
             arch: x86_64
-            runs: macos-12
+            runs: macos-13
```

Comment:
Nice! 

I think we should update the release workflow https://github.com/sketch-hq/fig2sketch/blob/40c326784f4c50f22662cb31e04b79ee13622438/.github/workflows/release.yml#L23 in this pull request too